Sources/Commands/SwiftPackageTool.swift

```diff
@@ -241,7 +242,13 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
         parser.add(subparser: PackageMode.clean.rawValue, overview: "Delete build artifacts")
         parser.add(subparser: PackageMode.fetch.rawValue, overview: "")
         parser.add(subparser: PackageMode.reset.rawValue, overview: "Reset the complete cache/build directory")
-        parser.add(subparser: PackageMode.update.rawValue, overview: "Update package dependencies")
+        let updateParser = parser.add(subparser: PackageMode.update.rawValue, overview: "Update package dependencies")
+
+        binder.bind(
+            option: updateParser.add(
+                option: "--package", kind: String.self,
+                usage: "The package name to update"),
+            to: { $0.updateOptions.packageName = $1 })
```

Comment:
What would you think about calling this `--only`?

Reply:
Further, what would you think about making this option a string *list*, so that a user could specify one or more specific package names to update?

Reply:
And what about not making it an option at all but just more arguments?
`swift package update` -> updates all
`swift package update dep1 dep2` -> updates just dep1 and dep2

Reply:
@ddunbar I can rename it. @jakepetroules @BenchR267 Maybe we can do that in another PR.